component.php

```diff
@@ -32,6 +32,28 @@ protected function get_random_group_id() {
 	}
 
 	/**
+	 * Generates a random user login
+	 *
+	 * @todo Improve for a more elegant solution.
+	 *
+	 * @since 1.3.0
+	 *
+	 * @param  int $length Length of the user login. Default: 6.
+	 * @return string
+	 */
+	protected function get_random_login( $length = 6 ) {
```

Comment:
@boonebgorges Do you have a better approach than this? This was the best I could think of.